fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -512,12 +512,16 @@ void complete(RelocateData const& relocation, std::map<UID, Busyness>& busymap,
 }
 
 // Cancells in-flight data moves intersecting with range.
-ACTOR Future<Void> cancelDataMove(struct DDQueue* self, KeyRange range, const DDEnabledState* ddEnabledState);
+ACTOR Future<Void> cancelDataMove(struct DDQueue* self,
+                                  KeyRange range,
+                                  const DDEnabledState* ddEnabledState,
+                                  UID debugID = UID());
 
 ACTOR Future<Void> dataDistributionRelocator(struct DDQueue* self,
                                              RelocateData rd,
                                              Future<Void> prevCleanup,
-                                             const DDEnabledState* ddEnabledState);
+                                             const DDEnabledState* ddEnabledState,
+                                             UID debugID = UID());
```

Comment:
Can we reuse the existing: `RelocateData::randomId`, instead of creating a new UID?

Reply:
RelocateData::randomId indicates any behavior of the same relocation, while debugID emphases the same context. DebugID is particularly useful when we want to see interleaves between a relocator and the cleanup triggered by the relocator.